app/Http/Controllers/PagesController.php

```diff
@@ -48,6 +51,12 @@ public function login(Request $request)
             $username = $request->input('username');
             $password = $request->input('password');
 
+            $user = User::where('username', $username)->first();
+
+            if ($user->confirmed == false) {
+                return \Redirect::back()->withErrors(trans('messages.notConfirmed'));
+            }
+
```

Comment:
I've pushed modifications. I check if username is not empty, to prevent bad messages (notConfirmed message instead of loginCombo message).